contracts/LpSugar.vy

```diff
@@ -1132,41 +1162,42 @@ def _cl_lp(_data: address[4], _token0: address, _token1: address) -> Lp:
   gauge_liquidity: uint128 = staticcall pool.stakedLiquidity()
   token0_fees: uint256 = 0
   token1_fees: uint256 = 0
-  locked: uint256 = staticcall locker_factory.locked(_data[1])
-  emerging: uint256 = staticcall self.cl_launcher.emerging(_data[1])
+  locked: uint256 = 0
+  emerging: uint256 = 0
   created_at: uint32 = 0
 
   slot: Slot = staticcall pool.slot0()
   tick_low: int24 = (slot.tick // tick_spacing) * tick_spacing
   tick_high: int24 = tick_low + tick_spacing
 
+  if self.cl_launcher.address != empty(address):
+    locker_factory: ILockerFactory = ILockerFactory(staticcall self.cl_launcher.lockerFactory())
```

Comment:
Should we cache the `locker_factory` to avoid fetching `lockerFactory()` when looping through the pools?